storage/src/main/java/tech/pegasys/teku/storage/server/kvstore/KvStoreConfiguration.java

```diff
@@ -99,7 +105,7 @@ public class KvStoreConfiguration {
   private CompressionType compressionType = CompressionType.LZ4_COMPRESSION;
 
   @JsonProperty("bottomMostCompressionType")
-  private CompressionType bottomMostCompressionType = CompressionType.NO_COMPRESSION;
+  private CompressionType bottomMostCompressionType = CompressionType.DISABLE_COMPRESSION_OPTION;
```

Comment:
I'm not sure why we want to change this. What are the benefits of changing the default to DISABLE_COMPRESSION_OPTION?

Reply:
i can change it, its configurable but it was using less cpu (and removal that you were testing is basically disable)

Reply:
i thougth we decided to merge this but still waiting on approval.

Reply:
we need to be clearer on these things because i thought we were merging this and have been waiting for this to be approved.

Reply:
that was before we had discussed in person... its fine its done.